### PR TITLE
Dependency updates

### PR DESCRIPTION
**Pull request by kmcnellis**
Reduce errors. Compiles via instructions on fresh ubuntu

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,6 +8,7 @@ dist
 /server/config/local.env.js
 npm-debug.log
 coverage
+yarn.lock
```
Comment (vakrao) [inline .gitignore]: Do we want to switch package mangaers to yarn?

**Comment by kmcnellis** [comment]
Yarn works alongside npm. Unfortunately, it didn't properly install all packages for me

**Comment by kmcnellis** [comment]
Either way, shouldn't be checked in to the repo

**Comment by seveibar** [comment]
`yarn.lock` is supposed to be checked into the repo

**Comment by seveibar** [comment]
hey everybody :)

**Comment by kmcnellis** [comment]
Ok, thanks

**Comment by vakrao** [comment]
rest of the updates seem good

**Pull request merged**